src/Microsoft.Health.Dicom.Metadata/Registration/Experimental/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Azure.Core;
+using Azure.Identity;
+using Azure.Storage.Blobs;
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Health.Blob.Configs;
+using Microsoft.Health.Dicom.Metadata;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IBlobServiceBuilder AddAzureBlobServiceClient(
+            this IServiceCollection services,
+            IConfiguration configurationRoot)
+        {
+            EnsureArg.IsNotNull(services, nameof(services));
+            EnsureArg.IsNotNull(configurationRoot, nameof(configurationRoot));
+
+            services.Configure<BlobContainerConfiguration>(
+                Constants.ContainerConfigurationName,
+                containerConfiguration => configurationRoot
+                    .GetSection("DicomWeb:MetadataStore")
+                    .Bind(containerConfiguration));
+
+            // TODO: Leverage Shared Components
+            var config = new BlobDataStoreConfiguration();
+            configurationRoot.GetSection("BlobStore").Bind(config);
```

Comment:
Question: how do you decide when to use Object.SectionName vs having a hard coded string?
I would guess that having it as an attribute would be better in most of the cases?

Reply:
I think SectionName is better, but BlobDataStoreConfiguration is defined in another repository, so...

Reply:
Section name is definitely better. Unfortunately this PR is not a collection of best practices, as I have temporarily copied chunks of code from the shared components to test this API. Once we get it to a place where we like it, then I'll merge it back